### ISSUE TITLE
3.0.0 Branch

### ISSUE THREAD

**Issue by nmccready** [angular-ui/angular-google-maps#1057]
@cthrax I know you have been working on getting the ball rolling on 3.0.0. I am just making this ticket to make this a public conversation. When you get the chance could you commit the branch when ready so I can start merging in develop changes.

Please post back the branch name when you get to it.

**Comment by cthrax** [comment]
Yeah, sorry. I expected to have more time over the holidays but came down with an illness that just wiped me out, I'll be traveling the next two weeks, but I'll see if I can get something up while I'm traveling.
